npm v7以上でnpm ciができない

npmパッケージ依存関係にconflictがあり、npm ciに失敗する

<details>
<summary>エラー</summary>

```text
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: webpack-extension-reloader@1.1.4
npm ERR! Found: webpack@5.68.0
npm ERR! node_modules/webpack
npm ERR!   dev webpack@"^5.68.0" from the root project
npm ERR!   peer webpack@"4.x.x || 5.x.x" from @webpack-cli/configtest@1.1.1
npm ERR!   node_modules/@webpack-cli/configtest
npm ERR!     @webpack-cli/configtest@"^1.1.1" from webpack-cli@4.9.2
npm ERR!     node_modules/webpack-cli
npm ERR!       dev webpack-cli@"^4.9.2" from the root project
npm ERR!       3 more (@webpack-cli/configtest, @webpack-cli/info, @webpack-cli/serve)
npm ERR!   7 more (copy-webpack-plugin, css-loader, html-webpack-plugin, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^4" from webpack-extension-reloader@1.1.4
npm ERR! node_modules/webpack-extension-reloader
npm ERR!   dev webpack-extension-reloader@"^1.1.4" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: webpack@4.46.0
npm ERR! node_modules/webpack
npm ERR!   peer webpack@"^4" from webpack-extension-reloader@1.1.4
npm ERR!   node_modules/webpack-extension-reloader
npm ERR!     dev webpack-extension-reloader@"^1.1.4" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/kikurage/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kikurage/.npm/_logs/2022-04-03T07_57_53_072Z-debug-0.log
```
</details>

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "typescript": "^4.5.5",
     "webpack": "^5.68.0",
     "webpack-cli": "^4.9.2",
-    "webpack-extension-reloader": "^1.1.4"
+    "webpack-ext-reloader": "^1.1.9"
   },
   "dependencies": {
     "@emotion/react": "^11.8.1",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,7 +1,7 @@
 const path = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
-const ExtensionReloader = require("webpack-extension-reloader");
+const ExtensionReloader = require("webpack-ext-reloader");
 
 module.exports = {
   mode: "development",

